docs/docs/segment-battery.md

```diff
@@ -42,5 +43,6 @@ Battery displays the remaining power percentage for your battery.
 - charged_color: `string` [color][colors] - color to use when fully charged - defaults to segment color
 - charging_color: `string` [color][colors] - color to use when charging - defaults to segment color
 - discharging_color: `string` [color][colors] - color to use when discharging - defaults to segment color
+- hide_while_charging: `bool` - Hide the battery status when it's plugged in(Charging or Full)
```

Comment:
Lower case `Hide`, add a space between `in` and `(` and add the default value (`false`).

Reply:
adapted with the new name